subsys/bluetooth/host/hci_core.c

```diff
@@ -802,6 +834,13 @@ static void le_conn_complete(struct net_buf *buf)
 		}
 	}
 
+	if (BT_FEAT_LE_DLE(bt_dev.le.features)) {
```

Comment:
As a peripheral, if remote does not support slave initiated feature request, them this tries the DLE procedure; if peer does not support it will respond with unknown rsp.